### ISSUE TITLE
Would it help if I loan (well, temporarily purchase) you an RX 6400?

### ISSUE THREAD

**Issue by ahydronous** [Rem0o/FanControl.ADLX#29]
Hello @Rem0o 

I really love Fan Control, and I really dislike that due to AMDs API changes, it works poorly with Fan Control and needs an iffy plugin.

You mentioned that its really hard to debug without acces to an AMD GPU.
Would it help if I bought a cheap RX 6400, had it shipped to you, and after you're done debugging you just ship it back to me (or sell it off and transfer me the money, if shipping costs would be egregious)?

**Comment by Rem0o** [comment]
Would be better of with an RDNA 3 card, as this is what ADLX was designed for in the first place. 

See https://gpuopen.com/introducing-adlx/

The reality of it is that for this to work there need to be a bit of a community effort. I made a big first step/push but haven't seen much interest nor expertise being added in.  I got a bit too much on my plate right now to dig in this specific project.

Also manual control seems to be just straight broken in the driver, or badly supported. The only workaround is editing the built-in graph, which is costly resource-wise and not viable (this is what the plugin does right now).  Haven't seen much development on the binding side either from AMD.   Until there is some kind of breakthrough or news from AMD, I won't put time onto this project.
